### PR TITLE
Initial-Routes / Code from workshop!

### PR DESCRIPTION
**Pull request by hellomkreyes**
## What Changed & Why?
1. `TaggedResults`: route/page displaying article results based on queried tag
1. `ArticlePreview`: component to display individual article information

## Screenshots / Videos
|Screenshot in Chrome|
|--|
|Screen Shot 2020-01-15 at 9 51 57 AM|

## Testing Notes
N/A

### Checklist
- [x] Functionally and visually tested in Firefox, Safari, and Chrome
- [x] No console warnings or errors

### CODE REVIEW

File: src/core/Routes.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { Switch, Route } from 'react-router-dom'
 
 import { HelloWorld } from './HelloWorld'
+import { TaggedResults } from './TaggedResults/TaggedResults'
```
Comment (hellomkreyes) [inline src/core/Routes.js]: For some reason, I get an error when trying to reference JUST `./TaggedResults`. Am I missing an export somewhere?

**Comment by hellomkreyes** [comment]
I have 1 warning because of the `Read More` anchor not having a valid href destination. Is there a way to easily mitigate this warning during development?

File: src/common/components/ArticlePreview/ArticlePreview.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const ArticlePreview = ({ article }) => (
```
Comment (mshelley-twg) [inline src/common/components/ArticlePreview/ArticlePreview.js]: Please add prop types :)

File: src/core/Routes.js (hunk 1 above)
Comment (mshelley-twg) [reply src/core/Routes.js]: What's the error?

Sometimes the compiler / watch process will fail when you move files around. You can try shutting it down, and restarting the process.

Comment (hellomkreyes) [reply src/core/Routes.js]: Oh wow, yeah. Restarting fixed it 😅

**Pull request merged**